spec/features/indexing_custom_component_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Indexing Custom Component', type: :feature do
+  let(:components) do # an Array<Arclight::CustomComponent>
+    options = { component: Arclight::CustomComponent }
+    ENV['SOLR_URL'] = 'http://127.0.0.1:8983/solr/blacklight-core'
```

Comment:
Yeah, `solr_ead` requires a Solr connection during its initialization. We could also try to put in a PR that does a lazy eval for the Solr connection. Rather than the `SOLR_URL` env varaible, it will take a `solr.yml`. Not sure which is the better approach.